WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
I'm sure I've seen it somewhere before, but I can't find it. Maybe it was actually the WCIF document.

Anyway...

1. I understand the motivation for using an integer representation of "centiseconds". If you use floats you'll constantly fight binary/decimal fuzziness, and end up with 9.31999947 when you _really_ need 9.32.

2. But it's really ugly, especially in a public API.

3. So I'd prefer that it's a 2 decimal number, at least in the WCIF. Since JSON is a string format, that shouldn't be that hard.

4. Not that I've thought through how to handle every issue around it.

5. In hindsight, a numbered list is a weird way to say this, but I'm not backing down now!